helm-frontend/src/components/Landing/SEAHELMLanding.tsx

```diff
@@ -9,6 +9,15 @@ export default function SEAHELMLanding() {
       </h1>
       <div className="flex flex-col lg:flex-row gap-8">
         <div className="flex-1 text-l">
+          <div className="text-center">
+            <a href="https://aisingapore.org/">
+              <img
```

Comment:
Unfortunately Stanford no longer permits us to put logos on the HELM website due to the [name use guidelines](https://trademarks.stanford.edu/name-use-guidelines), so I'll have to remove this in a subsequent pull request.